### DISCUSSION TITLE
Kata modulo 2

### DISCUSSION THREAD

**Discussion by luisviveros** [LaunchX-InnovaccionVirtual/CursoIntroPython#263] [General]
Hola buen dia a todos queria saber si me podrian decir como van a entregar o entregaron la kata modulo 2?

Les agradezco de antemano muchas gracias

**Comment by OscarPedroza** [comment]
Hola, puedes entregarla con capturas de pantalla en un notebook o un un pdf.

**Reply by luisviveros** [reply]
Muchas gracias Oscar
